packages/components/src/slot-fill/test/__snapshots__/slot.js.snap

```diff
@@ -155,13 +155,21 @@ exports[`Slot should render in expected order 1`] = `
   <div>
     first
     second
+    third
```

Comment:
This test fails at the moment with both the old implementation and the revised approach that ignores the concept of `occurrences`.